source/imgui_widgets.cpp

```diff
@@ -345,7 +349,11 @@ bool reshade::imgui::file_input_box(const char *name, std::filesystem::path &pat
 
 	return res;
 }
-bool reshade::imgui::directory_input_box(const char *name, std::filesystem::path &path, std::filesystem::path &dialog_path)
+bool reshade::imgui::directory_input_box(const char* name, std::filesystem::path& path, std::filesystem::path& dialog_path)
```

Comment:
It appears I had a wrong setting in the `.editorconfig` file which messed up the pointer/reference symbol alignment here (and elsewhere). Sorry about that, fixed now in https://github.com/crosire/reshade/commit/101c479057a0417960581e38aaa08ee449cb74e4. Mind adjusting these accordingly?